examples/Identifying Local Environments in a Complex Crystal.ipynb

```diff
@@ -24,7 +24,9 @@
    "cell_type": "code",
```

Comment:
<p>Let's use <code>gsd.hoomd.Frame</code> instead of <code>gsd.hoomd.Snapshot</code> so we don't get this deprecation warning in the jupyter notebook output.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/64/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>